packages/map/src/index.ts

```diff
@@ -1,7 +1,7 @@
 import { Accessor, batch } from "solid-js";
 import { TriggerCache } from "@solid-primitives/trigger";
 
-const $KEYS = Symbol("track-keys");
+const $OBJECT = Symbol("track-object");
```

Comment:
Changing to object as now we need to track object keys and values in both caches